### ISSUE TITLE
"ZeroDivisionError: float division" when smallest_count == largest_count

### ISSUE THREAD

**Issue by rasca** [niconoe/django-nuages-tag#2]
The traceback and the locals follows:

```
Traceback (most recent call last):
  File "/home/ivan/Sites/adoora/bootstrap/lib/python2.6/site-packages/django/contrib/staticfiles/handlers.py", line 67, in __call__
    return self.application(environ, start_response)
  File "/home/ivan/Sites/adoora/bootstrap/lib/python2.6/site-packages/django/contrib/staticfiles/handlers.py", line 67, in __call__
    return self.application(environ, start_response)
  File "/home/ivan/Sites/adoora/bootstrap/lib/python2.6/site-packages/django/core/handlers/wsgi.py", line 241, in __call__
    response = self.get_response(request)
  File "/home/ivan/Sites/adoora/bootstrap/lib/python2.6/site-packages/django/core/handlers/base.py", line 179, in get_response
    response = self.handle_uncaught_exception(request, resolver, sys.exc_info())
  File "/home/ivan/Sites/adoora/bootstrap/lib/python2.6/site-packages/django/core/handlers/base.py", line 111, in get_response
    response = callback(request, *callback_args, **callback_kwargs)
  File "/home/ivan/Sites/adoora/bootstrap/src/django-cms/cms/views.py", line 113, in details
    return render_to_response(template_name, context)
  File "/home/ivan/Sites/adoora/bootstrap/lib/python2.6/site-packages/django/shortcuts/__init__.py", line 20, in render_to_response
    return HttpResponse(loader.render_to_string(*args, **kwargs), **httpresponse_kwargs)
  File "/home/ivan/Sites/adoora/bootstrap/lib/python2.6/site-packages/django/template/loader.py", line 171, in render_to_string
    return t.render(Context(dictionary))
  File "/home/ivan/Sites/adoora/bootstrap/lib/python2.6/site-packages/django/template/base.py", line 140, in render
    return self._render(context)
  File "/home/ivan/Sites/adoora/bootstrap/lib/python2.6/site-packages/django/test/utils.py", line 62, in instrumented_test_render
    return self.nodelist.render(context)
  File "/home/ivan/Sites/adoora/bootstrap/lib/python2.6/site-packages/django/template/base.py", line 823, in render
    bit = self.render_node(node, context)
  File "/home/ivan/Sites/adoora/bootstrap/lib/python2.6/site-packages/django/template/debug.py", line 74, in render_node
    return node.render(context)
  File "/home/ivan/Sites/adoora/bootstrap/lib/python2.6/site-packages/django/template/loader_tags.py", line 123, in render
    return compiled_parent._render(context)
  File "/home/ivan/Sites/adoora/bootstrap/lib/python2.6/site-packages/django/test/utils.py", line 62, in instrumented_test_render
    return self.nodelist.render(context)
  File "/home/ivan/Sites/adoora/bootstrap/lib/python2.6/site-packages/django/template/base.py", line 823, in render
    bit = self.render_node(node, context)
  File "/home/ivan/Sites/adoora/bootstrap/lib/python2.6/site-packages/django/template/debug.py", line 74, in render_node
    return node.render(context)
  File "/home/ivan/Sites/adoora/bootstrap/lib/python2.6/site-packages/classytags/core.py", line 102, in render
    return self.render_tag(context, **kwargs)
  File "/home/ivan/Sites/adoora/bootstrap/lib/python2.6/site-packages/sekizai/templatetags/sekizai_tags.py", line 75, in render_tag
    rendered_contents = nodelist.render(context)
  File "/home/ivan/Sites/adoora/bootstrap/lib/python2.6/site-packages/django/template/base.py", line 823, in render
    bit = self.render_node(node, context)
  File "/home/ivan/Sites/adoora/bootstrap/lib/python2.6/site-packages/django/template/debug.py", line 74, in render_node
    return node.render(context)
  File "/home/ivan/Sites/adoora/bootstrap/lib/python2.6/site-packages/classytags/core.py", line 102, in render
    return self.render_tag(context, **kwargs)
  File "/home/ivan/Sites/adoora/bootstrap/lib/python2.6/site-packages/sekizai/templatetags/sekizai_tags.py", line 75, in render_tag
    rendered_contents = nodelist.render(context)
  File "/home/ivan/Sites/adoora/bootstrap/lib/python2.6/site-packages/django/template/base.py", line 823, in render
    bit = self.render_node(node, context)
  File "/home/ivan/Sites/adoora/bootstrap/lib/python2.6/site-packages/django/template/debug.py", line 74, in render_node
    return node.render(context)
  File "/home/ivan/Sites/adoora/bootstrap/lib/python2.6/site-packages/django/template/loader_tags.py", line 62, in render
    result = block.nodelist.render(context)
  File "/home/ivan/Sites/adoora/bootstrap/lib/python2.6/site-packages/django/template/base.py", line 823, in render
    bit = self.render_node(node, context)
  File "/home/ivan/Sites/adoora/bootstrap/lib/python2.6/site-packages/django/template/debug.py", line 74, in render_node
    return node.render(context)
  File "/home/ivan/Sites/adoora/bootstrap/lib/python2.6/site-packages/django/template/defaulttags.py", line 476, in render
    output = self.nodelist.render(context)
  File "/home/ivan/Sites/adoora/bootstrap/lib/python2.6/site-packages/django/template/base.py", line 823, in render
    bit = self.render_node(node, context)
  File "/home/ivan/Sites/adoora/bootstrap/lib/python2.6/site-packages/django/template/debug.py", line 74, in render_node
    return node.render(context)
  File "/home/ivan/Sites/adoora/bootstrap/lib/python2.6/site-packages/classytags/core.py", line 102, in render
    return self.render_tag(context, **kwargs)
  File "/home/ivan/Sites/adoora/bootstrap/src/django-cms/cms/templatetags/cms_tags.py", line 223, in render_tag
    content = get_placeholder_content(context, request, page, name, inherit)
  File "/home/ivan/Sites/adoora/bootstrap/src/django-cms/cms/templatetags/cms_tags.py", line 145, in get_placeholder_content
    content = render_placeholder(placeholder, context, name)
  File "/home/ivan/Sites/adoora/bootstrap/src/django-cms/cms/plugin_rendering.py", line 127, in render_placeholder
    c.extend(render_plugins(plugins, context, placeholder, processors))
  File "/home/ivan/Sites/adoora/bootstrap/src/django-cms/cms/plugin_rendering.py", line 81, in render_plugins
    c.append(plugin.render_plugin(context, placeholder, processors=processors))
  File "/home/ivan/Sites/adoora/bootstrap/src/django-cms/cms/models/pluginmodel.py", line 183, in render_plugin
    return render_plugin(context, instance, placeholder, template, processors)
  File "/home/ivan/Sites/adoora/bootstrap/src/django-cms/cms/plugin_rendering.py", line 53, in render_plugin
    content = render_to_string(template, context)
  File "/home/ivan/Sites/adoora/bootstrap/lib/python2.6/site-packages/django/template/loader.py", line 171, in render_to_string
    return t.render(Context(dictionary))
  File "/home/ivan/Sites/adoora/bootstrap/lib/python2.6/site-packages/django/template/base.py", line 140, in render
    return self._render(context)
  File "/home/ivan/Sites/adoora/bootstrap/lib/python2.6/site-packages/django/test/utils.py", line 62, in instrumented_test_render
    return self.nodelist.render(context)
  File "/home/ivan/Sites/adoora/bootstrap/lib/python2.6/site-packages/django/template/base.py", line 823, in render
    bit = self.render_node(node, context)
  File "/home/ivan/Sites/adoora/bootstrap/lib/python2.6/site-packages/django/template/debug.py", line 74, in render_node
    return node.render(context)
  File "/home/ivan/Sites/adoora/bootstrap/lib/python2.6/site-packages/django_nuages_tag/templatetags/django_nuages_tag.py", line 50, in render
    self.max_size, self.min_size)
  File "/home/ivan/Sites/adoora/bootstrap/lib/python2.6/site-packages/django_nuages_tag/templatetags/django_nuages_tag.py", line 63, in calculate_lin
    -smallest_count) ) + min_size
ZeroDivisionError: float division


>>> locals()
{'min_size': 10.0, 'largest_count': 9, 'current_count': 9, 'smallest_count': 9, 'max_size': 25.0  }
```

**Comment by niconoe** [comment]
Thanks for reporting this bug Rasca, it will be fixed very soon !

**Comment by niconoe** [comment]
This has been fixed, and version a corrected 0.1.3 version has been released. Feel free to install it via pip and report any other issue.

Thx again !

**Comment by rasca** [comment]
@niconoe thank _you_!

I've already tested it and it's working fine, thanks again!
